src/ASM/LR/ASMu3Dmx.C

```diff
@@ -178,6 +178,17 @@ bool ASMu3Dmx::generateFEMTopology ()
     m_basis.resize(vec.size());
     for (size_t i=0;i<vec.size();++i)
       m_basis[i].reset(new LR::LRSplineVolume(vec[i].get()));
+
+    // we need to project on something that is not one of our bases
+    if (ASMmxBase::Type == ASMmxBase::REDUCED_CONT_RAISE_BASIS1 ||
+        ASMmxBase::Type == ASMmxBase::DIV_COMPATIBLE ||
+        ASMmxBase::Type == ASMmxBase::SUBGRID) {
+      auto vec2 = ASMmxBase::establishBases(tensorspline,
+                                            ASMmxBase::FULL_CONT_RAISE_BASIS1);
+      projBasis.reset(new LR::LRSplineVolume(vec2.front().get()));
```

Comment:
See previous comment on the same issue for 2D

Reply:
i gave it the same treatment as 2D. it will be like 2D once i get around to implementing 3D mixed refine.